### PR TITLE
set up configurations for development on v5

### PR DESCRIPTION
**Pull request by nicolethoen**
- setting up renovate bot to pull in the core v5 branch when it releases an alpha
- updating the packages so they'll publish to `patternfly-five` npm org with alpha tags
- setting up the release script so that it'll create and update patternfly-react-v5.surge.sh everytime a pr is merged into v5

Note: This won't work until we merge it. We'd need to merge it to really test it. It may require follow up PRs to clean up (it took about 5 PRs to get it all set up correctly in core).
All my notes for how to configure a development branch for major release work can be found [in this doc](https://docs.google.com/document/d/1aRpzrhWZO1R30MFCnH8FtLEw43ZOn_K_XpFYWKEQGZs/edit#) and I will update it as we learn.

### CODE REVIEW

File: packages/react-topology/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@patternfly/react-topology",
+  "name": "@patternfly-react/react-topology",
```
Comment (evwilkin) [inline packages/react-topology/package.json:2]: Should this also be `@patternfly-five`?
```suggestion
  "name": "@patternfly-five/react-topology",
```

File: packages/react-log-viewer/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@patternfly/react-log-viewer",
+  "name": "@patternfly-react/react-log-viewer",
```
Comment (evwilkin) [inline packages/react-log-viewer/package.json:2]: Should this be `@patternfly-five`?
```suggestion
  "name": "@patternfly-five/react-log-viewer",
```

**Review by kmcfaul — APPROVED** [review]

**Review by evwilkin — APPROVED** [review]
LGTM

File: packages/react-code-editor/package.json [hunk 3]
```diff
@@ -30,10 +30,10 @@
     "clean": "rimraf dist"
   },
   "dependencies": {
-    "@patternfly/react-core": "^4.251.1",
-    "@patternfly/react-icons": "^4.92.6",
-    "@patternfly/react-styles": "^4.91.6",
-    "react-dropzone": "^14.2.3",
+    "@patternfly-react/react-core": "^1.0.0",
+    "@patternfly-react/react-icons": "^1.0.0",
+    "@patternfly-react/react-styles": "^1.0.0",
```
Comment (jenny-s51) [inline packages/react-code-editor/package.json:35]: Should these be `@patternfly-five` as well? It looks like the scope `@patternfly-react` doesn't exist on npm... When I install I get an error for each of those instances:

```
error Couldn't find package "@patternfly-react/react-core@^1.0.0" required by "@patternfly-five/react-code-editor@1.0.0" on the "npm" registry.
```

```suggestion
    "@patternfly-five/react-core": "^1.0.0",
    "@patternfly-five/react-icons": "^1.0.0",
    "@patternfly-five/react-styles": "^1.0.0",
```

**Review by evwilkin — APPROVED** [review]
LGTM

**Comment by evwilkin** [comment]
> LGTM

...minus the failed build 🤔

**Review by jenny-s51 — COMMENTED** [review]
Looks like we might need to update the `@patternfly/react-[pkg]` references to `@patternfly-five/react-[pkg]`?

**Comment by nicolethoen** [comment]
> Looks like we might need to update the `@patternfly/react-[pkg]` references to `@patternfly-five/react-[pkg]` throughout the repo?

I've been trying figure out the scope of that all morning - not super straight forward.

there are also dependencies/references to some patternfly packages in the documentation framework, so i'm wondering if we will need a v5 of the documentation-framework @dgutride @evwilkin

**Pull request closed without merging**